cartridge/graphql/execute.lua

```diff
@@ -213,6 +213,10 @@ local function completeValue(fieldType, result, subSelections, context, opts)
   end
 
   if fieldTypeName == 'Object' then
+    if type(result) ~= 'table' then
+      local message = ('Expected %q to be a "table", got %q'):format(fieldName, type(result))
```

Comment:
Also, to be consistent, I would change to pattern `'Expected %q to be a "map", got %q'`

Reply:
It's redundant I think. Because you should declare nested fields for map. And array won't have them. In contrast map was interpreted as an empty array thus is_array check makes sense.

Reply:
Agree, fixed